### DISCUSSION TITLE
pyMOR Community Meeting 2023-03-08

### DISCUSSION THREAD

**Discussion by pmli** [pymor/pymor#1967] [Announcements]
The next pyMOR Community Meeting will take place this Wednesday, March 8, at 14:00 CET and 8:00 EST, online at:

- link: https://bbb.mpi-magdeburg.mpg.de/meeting/pymor
- password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- release process (@HenKlei)
- pyMOR School 2023 (@HenKlei)
- quickstart document (@sdrave)
- pyMOR performance with NumpyMatrixOperator (@drittelhacker)
- [documentation system](https://diataxis.fr/) (@pmli)

**Comment by lbalicki** [comment]
### **Meeting Notes**

release process
- Should we switch release manager every release? Probably not necessary. We should decide before each release who will do the job.
- Set dates for soft/hard freeze/release date about 1 month before soft freeze date. Soft freeze starts 4 weeks before release day. Hard freeze starts 2 weeks before release day.
- Will update https://github.com/pymor/pymor/pull/1946/files

pyMOR school 2023
- Will discuss with AP in future

quickstart document
- short code snippet for running different MOR methods for different models in pyMOR
- first step could be a "benchmark collection"
- probably procedural benchmarks would be good for getting started
- make new submodule: pymor.models.examples

documentation system
- 4 different types of documentation: How-to, tutorial, explanation, reference
- The technical overview is similar to explanation and can possibly be restructured a little bit
- Need to update README.md

pyMOR performance with NumpyMatrixOperator
- using NumpyMatrixOperator based on .from_matrices apply_inverse is based on scipys spsolve
- umfpack is only used if scikit.umfpack is installed otherwise SuperLU
- put comment into README since umfpack is often faster
- open question: What about complex number support? What about storing LU factors?

Next Meeting: 2023-03-22, 13:00 UTC (14:00 CET, 9:00 EDT)
